packages/wrangler/scripts/bundle.ts

```diff
@@ -13,11 +14,16 @@ async function run() {
     format: "cjs",
     // minify: true, // TODO: enable this again
     external: ["fsevents", "esbuild", "miniflare", "@miniflare/core"], // todo - bundle miniflare too
-    sourcemap: true,
+    sourcemap: "external",
     inject: [path.join(__dirname, "../import_meta_url.js")],
     define: {
       "import.meta.url": "import_meta_url",
     },
+  }).then(() => {
+    appendFileSync(
+      "./wrangler-dist/cli.js",
+      `//# sourceMappingURL=/cli.js.map`
```

Comment:
If esbuild is set to `sourcemap: external` I am surprised that it does not automatically add this comment to the output...

Reply:
Oh I see "external" explicitly does not add the comment. I think we want "linked" here?
https://esbuild.github.io/api/#sourcemap

Reply:
Indeed it looks like the default when `sourcemap: true` is "external"
https://github.com/evanw/esbuild/blob/master/lib/shared/common.ts#L360

Reply:
I'm certain esbuild adds the line with `sourcemap:true`, so this addition shouldn't be necessary.

Reply:
I needed to be able to control what the actual URL was that it had in the comment. I will check if it works.

Reply:
The `/` should not be needed. SourceMap links via URLs are always relative to the file containing them. So

```
//# sourceMappingURL=cli.js.map
```

"should" be correct.

Can you demo the problem?

Reply:
> The `/` should not be needed. SourceMap links via URLs are always relative to the file containing them. So

It is in this case as far as I have been able to determine.

> ```
> //# sourceMappingURL=cli.js.map
> ```
> 
> "should" be correct.
> 
> Can you demo the problem?

Yes we can discuss in our next meeting.

Reply:
Handling the SourceMap scripting logic in GitHub Actions